nginx/config.go

```diff
@@ -186,6 +186,11 @@ http {
 		location / {
 			return 404;
 		}
+		{{ if $routerConfig.statusEndpoint }}
```

Comment:
I see you groomed the commits and force-pushed... I think you lost the model changes that complement this.